.vscode/remix.code-snippets

```diff
@@ -0,0 +1,76 @@
+{
+	"loader": {
+		"prefix": "/ loader",
```

Comment:
```suggestion
		"prefix": "/loader",
```

Reply:
Yes. In Visual Studio Code, for example, you can do this by adding a "scope" field with the desired language identifiers. 
```json
		"scope": "typescriptreact,javascriptreact,typescript,javascript",
```